extractor/main.go

```diff
@@ -105,10 +105,23 @@ func execExtract(ctx context.Context, cfg *extractorCfg) error {
 		return errInvalidOutputDir
 	}
 
-	// Find the files that need to be analyzed
-	sourceFiles, findErr := findFilePaths(cfg.sourceDir, cfg.fileType)
-	if findErr != nil {
-		return fmt.Errorf("unable to find file paths, %w", findErr)
+	var sourceFiles []string
+	var findErr error
```

Comment:
This error declaration is useless, you only utilize it in the `if` statement. Please make it scoped to that condition block instead

Reply:
Scoped it properly [748ef8d](https://github.com/gnolang/tx-exports/pull/11/commits/748ef8d27458e49eef9309678efdf71324bb52e8)